app/spaces/[spaceid]/components/sidebar/spaceSubSidebar/spaceSubSidebar.tsx

```
@@ -0,0 +1,194 @@
+        />
+      ));
+
+    return [
+      hasCalendar ? (
+        <TabItem
+          label="Calendar"
+          href={`/spaces/${spaceId}/calendar`}
+          icon={<CalendarDots />}
+          isActive={isRouteActive('calendar')}
+          onClick={onCloseDrawer}
+        />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)